utils/src/ooinstall/cli_installer.py

```diff
@@ -722,14 +722,30 @@ def upgrade(ctx):
         click.echo("No hosts defined in: %s" % oo_cfg.config_path)
         sys.exit(1)
 
-    # Update config to reflect the version we're targetting, we'll write
-    # to disk once ansible completes successfully, not before.
     old_variant = oo_cfg.settings['variant']
     old_version = oo_cfg.settings['variant_version']
```

Comment:
Totally makes sense. We should probably also query to figure out which versions they have available too so the user doesn't start an upgrade they can't actually finish.

But, this is just a change to our existing model to let them kick off a minor upgrade instead of assuming that they want to do a major upgrade.